Gopkg.toml

```diff
@@ -28,7 +28,10 @@ required = [
 
 [[override]]
   name = "knative.dev/pkg"
-  branch = "master"
+  #TODO(mattmoor): DO NOT SUBMIT
```

Comment:
<!--donotsubmit-bot-->
Found "DO NOT SUBMIT".

Reply:
haha cool. do you happen to have these tricks documented anywhere?

Reply:
Just the source: github.com/mattmoor/knobots.  This is one Google has internally, which I missed :)